linkerd/app/src/dst/resolve.rs

```diff
@@ -51,8 +48,8 @@ pub struct Unresolvable(());
 
 // === impl PermitNamesInSuffixes ===
 
-impl<I: IntoIterator<Item = Suffix>> From<I> for PermitNamesInSuffixes {
```

Comment:
FWIW, I think the trait we were looking for here was [`FromIterator`](https://doc.rust-lang.org/std/iter/trait.FromIterator.html).

Probably doesn't really matter, since this isn't a public API.